src/views/HelpAndKnowledge/SolutionsHelp/solutionsMap.tsx

```diff
@@ -1096,5 +1097,32 @@ export const helpSolutions: HelpSolutionType[] = [
         <SharedSystemsTimeLine {...props} />
       )
     }
+  },
+  {
+    enum: OperationalSolutionKey.BCDA,
```

Comment:
I think that you might need to add additional information in the `helpAndKnowledge.ts`
![image](https://github.com/CMSgov/mint-app/assets/99294987/c73c2238-3659-4979-9119-d8e12f4a7f29)

The description is missing when you look at this card, I also don't see the details in the help situation.
![image](https://github.com/CMSgov/mint-app/assets/99294987/abf314fe-1a6e-40c9-b2a0-9fad448fd4a4)
I think that  ts file is where you'd need to add it.

Reply:
We discussed this point during today's standup. @patrickseguraoddball is going to pick up the associated front-end ticket.